tier4_v2x_msgs/msg/ExtraTrafficSignal.msg

```diff
@@ -0,0 +1,7 @@
+std_msgs/Header header
+autoware_perception_msgs/TrafficLightGroup[] states
```

Comment:
Since this message is for describing the remaining time for traffic light state, it might be better to explicitly name the variable name as `current_state` or `next_state` since it is confusing for the users.